zulipterminal/model.py

```diff
@@ -656,31 +656,3 @@ def poll_for_events(self) -> None:
                 for event_type, do_action_for in self.event_actions.items():
                     if event_type == event['type']:
                         do_action_for(event)
-
-    def get_msg_info(self, msg: Dict[str, Any]) -> Dict[str, Any]:
-        msg_info = {
-            'DATE & TIME': time.ctime(msg['timestamp'])[:-5],
-            'SENDER': msg['sender_full_name'],
-            'SENDER\'S EMAIL ID': msg['sender_email'],
-            'RECIPIENT': msg['display_recipient'] + " -> " + msg['subject'] if
-                                                    msg['type'] == 'stream'
-                                                    else
-            msg['display_recipient'][0]['full_name'] if not
-            msg['display_recipient'][0]['full_name'] == msg['sender_full_name']
-            else 'Self',
-            'REACTIONS': sorted(
-                    [reaction['emoji_name'] +
-                        ": " +
-                        reaction['user']['full_name'] +
-                        "\n"
-                        if 'user_id' in reaction['user']
-                        else reaction['emoji_name'] +
-                        ": " +
-                        reaction['user']['full_name'] +
-                        "\n"
-                        for reaction in msg['reactions']])
-            if msg['reactions']
-            else '---None---',
-            # 'EDIT HISTORY':
-            }
-        return msg_info
```

Comment:
It seems a bit strange to introduce this in the first commit and then remove it in this commit?

Also, if you can make this commit just about adding the edit history, that would probably be sufficient for one commit.